src/frontend/src/styles/main.css

```diff
@@ -1,676 +1,820 @@
+/* revert the 'white-space' property for textarea elements on Safari */
+textarea {
+  white-space: revert;
+}
 
-  line-height: 1.25rem;
-  letter-spacing: 0.005em;
+/* minimum style to allow to style meter element */
+meter {
+  -webkit-appearance: revert;
+  appearance: revert;
 }
 
-input.errored {
-  border: double 2px var(--flamingo-500);
+/* reset default text opacity of input placeholder */
+::placeholder {
+  color: unset;
 }
-input.errored::placeholder {
-  color: var(--flamingo-500);
+
+/* fix the feature of 'hidden' attribute.
+display:revert; revert to element instead of attribute */
+:where([hidden]) {
+  display: none;
 }
 
-.error-message-hidden {
-  visibility: hidden;
-  height: 0;
-  margin: 0;
+/* revert for bug in Chromium browsers
+- fix for the content editable attribute will work properly.
+- webkit-user-select: auto; added for Safari in case of using user-select:none on wrapper element*/
+:where([contenteditable]:not([contenteditable="false"])) {
+  -moz-user-modify: read-write;
+  -webkit-user-modify: read-write;
+  overflow-wrap: break-word;
+  -webkit-line-break: after-white-space;
+  -webkit-user-select: auto;
 }
 
-.error-message {
-  color: var(--flamingo-500);
-  margin: 0 0 0.5rem;
-  font-size: 1rem;
+/* apply back the draggable feature - exist only in Chromium and Safari */
+:where([draggable="true"]) {
+  -webkit-user-drag: element;
 }
 
-textarea {
-  width: 100%;
-  height: 6rem;
-  box-sizing: border-box;
-  margin-bottom: 1rem;
-  font-weight: 400;
-  font-family: inherit;
-  resize: none;
-  padding: 0.3rem;
-  text-align: center;
+/**
+ * IV) Base: Un-classed HTML elements (type selectors)
+ *
+ * Internet Identity Styles
+ **/
+
+:root {
+  /* general styles */
+  font-family: "Montserrat", sans-serif;
+  font-size: max(62.5%, calc(17.5% + 0.6vw)); /* 10px / 1200px min 10px */
+
+  background: var(--rc-background);
+  color: var(--rc-text);
 }
 
-h1 {
-  margin-top: 0;
-  font-size: 1.8rem;
-  margin: auto;
+body {
+  font-size: 1.6rem;
+  line-height: var(--vs-line-height);
 }
-@media (min-width: 1024px) {
-  h1 {
-    margin: inherit;
-  }
+
+/**
+ * V) Objects: Cosmetic-free design patterns
+ * 
+ * We generally use the BEM methodology.
+ * http://getbem.com/
+ *
+ * We prefix classes with the following convention:
+ * - `.c-` component
+ * - `.l-` layout
+ * - `.t-` typography
+ *
+ * We avoid element, id and attribute selectors when it is saint to do so. 
+ * Use classes when possible. 
+ */
+
+.t-strong {
+  font-weight: bold;
 }
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-p {
-  margin: 0 0 1rem;
+.t-weak {
+  font-weight: normal;
+  font-size: 0.8em;
 }
 
-details summary {
-  cursor: pointer;
+.t-title {
+  font-weight: bold;
+  font-size: 1.8rem;
+  line-height: 1.1;
+  margin-top: 1em;
 }
 
-summary::marker {
-  content: none;
+.t-title:first-child {
+  margin-top: 0;
 }
 
-button {
-  border: 1px solid var(--grey-200);
-  background: var(--background-color);
+.t-title--complications {
```

Comment:
one is a modifier the other one a child `--` sais "they this element has complication" and the other one is a single complication that is a child of the parent element. We avoid using `title--complications__complication` because a complication it self could also have a modifier in the future, that would end up creating things like `title--complications__complication--modifierofthechild`. I hope this answers your question: http://getbem.com/faq/#can-a-block-modifier-affect-elements-